components/common/LoginForm.jsx

```diff
@@ -25,13 +30,31 @@ const LoginForm = () => {
     const response = await customAPICall('auth/login', 'post', { email, password });
     if (response?.success) {
       if (response.data?.access_token) {
+        // Call to get permissions
         dispatch(setToken({ token: response.data.access_token }));
-        // Setting Token Expiry
-        dispatch(
-          setTokenExpireTime({ tokenExpireTime: Date.now() + response.expires_in * 1000 })
-        );
-        sendToast('success', 'Login Successful', 4000);
-        router.push('/');
+        const me = await customAPICall('auth/me', 'post');

```

Comment:
I've already included user object in response to login, so you don't need to make auth/me call anymore on login